pages/claimstatus/index.tsx

```diff
@@ -18,16 +35,87 @@ export default function Home(): ReactElement {
         <title>{t('title')}</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
-
       <Header />
       <Main />
       <Footer />
+      {console.log("-Test Response-")} {/* testing, to be removed before launch */}
+      {console.dir({claimData})} {/* testing */}
     </Container>
   )
 }
 
-export const getServerSideProps: GetServerSideProps = async ({ locale }) => ({
-  props: {
-    ...(await serverSideTranslations(locale || 'en', ['common', 'header', 'footer'])),
-  },
-})
+export const getServerSideProps: GetServerSideProps = async ({ req, locale }) => {
+  const PASSWORD = process.env.CERTIFICATE_PASSPHRASE
+  const API_URL: string = process.env.API_URL || "fail gracefully" //TODO: Fail gracefully. 
```

Comment:
I think you can create a separate ticket for "implement error and alert when any critical environment variable does not exist", point to these lines, and move on for this PR. if the env vars for the API Gateway don't exist, that's an urgent issue.

Reply:
https://github.com/cagov/ui-claim-tracker/issues/176